toolchain/sem_ir/name_scope.h

```diff
@@ -51,6 +51,7 @@ class NameScope : public Printable<NameScope> {
 
   // Searches for the given name and returns an EntryId if found or nullopt if
   // not.
+  // If the name is found, it might be a poisoned name.
```

Comment:
Generally I'd suggest writing comments like you would markdown:

- Put a blank line inbetween if you want separate paragraphs. i.e.:

```suggestion
  // not.
  //
  // If the name is found, it might be a poisoned name.
```

- Wrap if you want a single paragraph:

```suggestion
  // not. If the name is found, it might be a poisoned name.
```

- Use bullets (`// - text here`) if you're trying to list things. (probably doesn't apply in this comment)

Just noting this for style of below comments too.

Reply:
Done.